### ISSUE TITLE
[BUG] SyncTeamSpeakInBackground returned a NULL offset

### ISSUE THREAD

**Issue by NNMavy** [nathan-fiscaletti/TeamSpeak-Integration-Issue-Tracker#5]
**Do you own the TS3 Integration application?**
Yes, https://invisioncommunity.com/profile/573278-mavy/

**Describe the bug**
The background sync task thrown an error in the IPS system log. The full message is
`SyncTeamSpeakInBackground returned a NULL offset - tasks should throw \IPS\Task\Queue\OutOfRangeException when they are finished`

**To Reproduce**
In the Admincp enable the "Auto Sync Groups - Task" option. Wait for a sync to run as a background task and the log will be in the System Log

**Expected behavior**
No errors in the system log when the task runs successfully.

**Additional context**
The backtrace lists the following:
```
#0 /var/www/live/system/Task/Task.php(55): IPS\_Log::log('SyncTeamSpeakIn...', 'runQueue_log')
#1 /var/www/live/applications/core/tasks/queue.php(43): IPS\_Task::runQueue()
#2 /var/www/live/system/Task/Task.php(356): IPS\core\tasks\_queue->IPS\core\tasks\{closure}()
#3 /var/www/live/applications/core/tasks/queue.php(55): IPS\_Task->runUntilTimeout(Object(Closure))
#4 /var/www/live/system/Task/Task.php(255): IPS\core\tasks\_queue->execute()
#5 /var/www/live/system/Task/Task.php(224): IPS\_Task->run()
#6 /var/www/live/applications/core/interface/task/task.php(62): IPS\_Task->runAndLog()
#7 {main}
```

**Comment by nathan-fiscaletti** [comment]
This has been fixed in https://github.com/nathan-fiscaletti/ts3integration/commit/849fc32ca43635d59703b9c4159fc516e341b7de and will be merged soon and released with v2.0.7. This issue will remain open until the merge closes it.

**Comment by nathan-fiscaletti** [comment]
Fixed in Version 2.0.7
